### PR TITLE
Add NONUSE option for condor in startup

### PR DESCRIPTION
**Pull request by manabuishii**
supervisor mange condor.

### CODE REVIEW

File: templates/startup.sh.j2 [hunk 1]
```diff
@@ -125,6 +125,13 @@ function start_supervisor {
         supervisorctl start galaxy:galaxy_nodejs_proxy
     fi
     {% endif %}
+    {% if supervisor_manage_condor|bool %}
+            if [[ $NONUSE != *"condor"* ]]
```
Comment (bgruening) [inline templates/startup.sh.j2]: Can you please fix the indentation here?

**Comment by bgruening** [comment]
Looks great! Thanks!

**Comment by manabuishii** [comment]
@bgruening  Is this ok for indent ?

**Comment by bgruening** [comment]
Yes, this is super!

**Comment by manabuishii** [comment]
Thank you!
I check the galaxy-stable dev and I confirm that -e "NONUSE=condor" works.
Thank you for your quick merge and update galaxy-stable dev. :smile:

**Comment by bgruening** [comment]
@manabuishii your welcome! Thanks to you for this nice contribution! Btw. I branched the 16.07 release - if you want to test the next upcoming stable release. It should have your patch integrated as well.

**Pull request merged**